Vungle/VungleRouter.m

```diff
@@ -454,22 +454,23 @@ - (void)completeBannerAdViewForPlacementID:(NSString *)placementID
 - (void)invalidateBannerAdViewForPlacementID:(NSString *)placementID
                                     delegate:(id<VungleRouterDelegate>)delegate
 {
-    @synchronized (self) {
+    __weak VungleRouter *weakself = self;
+    dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
@YueVungle Is it no problem to use "main_queue" for sync process here (instead of "synchronize" or background sync queue)? Just curious.

Reply:
@ashinagawa Hi Aki, I used @synchronized (self) to synchronize before, but on iOS 13, this way doesn't guarantee thread safety and lead to crash(I don't know if it is a system bug), so I use main queue to do thread synchronization. Because some functions (have UI operations) must be guaranteed to run on the main thread, so I did not use background queue for thread synchronization. Thanks a lot. ^_^

Reply:
@YueVungle Thanks for clarifying. Got it. Please go ahead and merge it.
In future, we might want to do all adapter process in background, and only when we send "callbacks" to mediation SDK, we can make sure that we send them in main queue.
But, that'd be future adapter improvement.

Reply:
@ashinagawa Hi Aki, got it, thanks a lot for the detailed explanation. ^_^